app/views/account/IsraelSignupView.coffee

```diff
@@ -0,0 +1,142 @@
+RootView = require 'views/core/RootView'
+template = require 'templates/account/israel-signup-view'
+forms = require 'core/forms'
+errors = require 'core/errors'
+utils = require 'core/utils'
+User = require 'models/User'
+
+class AbortError extends Error
+
+formSchema =
+  type: 'object'
+  properties: _.pick(User.schema.properties, 'name', 'password')
+  required: ['name', 'password']
+
+
+module.exports = class IsraelSignupView extends RootView
+  id: 'israel-signup-view'
+  template: template
+  
+  events:
+    'submit form': 'onSubmitForm'
+    'input input[name="name"]': 'onChangeName'
+    'input input[name="password"]': 'onChangePassword'
+    
+  initialize: ->
+    @state = new Backbone.Model({
```

Comment:
Well, I didn't really need anything special in a State object. I went for the simplest tool. For consistency, though, should switch.